desk/bare/web/fund/page/proj-edit.hoon

```diff
@@ -103,12 +111,11 @@
                 ;textarea.p-1  =name  "sum"  =rows  "3"
                   =placeholder  "Write a worthy description of your project"
                   =value  (trip ?~(pru '' summary.u.pru))
-                  ::  FIXME: Ghastly, but needed for auto-grow trick (see fund.css)
-                  =oninput  "this.parentNode.dataset.replicatedValue = this.value"
+                  =x-on-input  "updateTextarea"
                   ; {(trip ?~(pru '' summary.u.pru))}
                 ==
               ==
-              ;label(for "sum"): project description
+              ;label(for "sum"): project description (markdown)
```

Comment:
Instead of adding this to the label, let's call out in the placeholder text that this field supports markdown.